### ISSUE TITLE
Liberty 19.0.0.7 Blocks *all* Large Object API functions for Postgres

### ISSUE THREAD

**Issue by sshivell** [OpenLiberty/open-liberty#8346]
The Postgres JDBC Driver allows for accessing the LargeObjectAPI calls to deal with storing large files with the `oid` type in Postgres.

As of Websphere Liberty 19.0.0.7 these apis have been blocked for use by Websphere.

Attempting to use them results in errors such as:
```
java.sql.SQLFeatureNotSupportedException: DSRA9130E: Operation is not permitted by the application server: getLargeObjectAPI
    at com.ibm.ws.rsadapter.jdbc.WSJdbcWrapper.invoke(WSJdbcWrapper.java:224)
    at com.ibm.ws.rsadapter.jdbc.WSJdbcConnection.invoke(WSJdbcConnection.java:4124)
    at 
```
My understanding, based on conversations with devs and after seeing some Tests written in this issue: https://github.com/OpenLiberty/open-liberty/issues/8118  Is that  the LargeObjectAPI with postgres can do things to the underlying Connection outside of WebSphere's Transaction manager.  The end result is that some operation can result in data-integrity issues.

However, not all uses of the api are unsafe and some are safe.    Asking that safe operations not be blocked. Specifically things like:
`conn.largeObjectApi.createLO()`
`conn.largeObjectApi.open(int,int)`  which is
```
  public LargeObject open(long oid, int mode) throws SQLException {
    return open(oid, mode, false);
  }
```
LargeObject..getOutputStream()
```
@throws SQLException if a database-access error occurs.
   */
  public OutputStream getOutputStream() throws SQLException {
    if (os == null) {
      os = new BlobOutputStream(this, 4096);
    }
    return os;
  }
```

**Comment by aguibert** [comment]
thanks for reporting this @sshivell. Originally I had blocked the entire LargeObjectAPI for 19.0.0.7 because some of the operations could manipulate the underlying connection which would go beyond Liberty's transaction manager tracking.

However, after revisiting this issue I've realized a way that Liberty can reflect/proxy the API call in such a way that the tx manager maintains tracking of the underlying connection in PR #8351. I expect this limitation will be lifted in the next release (19.0.0.8).
